specification/webpubsub/data-plane/WebPubSub/stable/2023-07-01/webpubsub.json

```diff
@@ -0,0 +1,1821 @@
+            "schema": {
+              "$ref": "#/definitions/ErrorDetail"
+            },
+            "x-ms-error-response": true,
+            "headers": {
+              "x-ms-error-code": {
+                "x-ms-client-name": "ErrorCode",
+                "type": "string"
+              }
+            }
+          }
+        },
+        "x-ms-examples": {
+          "WebPubSub_RemoveUserFromGroup": {
+            "$ref": "./examples/WebPubSub_RemoveUserFromGroup.json"
+          }
+        }
+      },
+      "put": {
+        "tags": [
+          "webpubsub"
+        ],
+        "summary": "Add a user to the target group.",
+        "operationId": "WebPubSub_AddUserToGroup",
+        "produces": [
+          "application/json"
+        ],
+        "parameters": [
+          {
+            "in": "path",
+            "name": "hub",
+            "description": "Target hub name, which should start with alphabetic characters and only contain alpha-numeric characters or underscore.",
+            "required": true,
+            "type": "string",
+            "pattern": "^[A-Za-z][A-Za-z0-9_`,.[\\]]{0,127}$"
+          },
+          {
+            "in": "path",
+            "name": "group",
+            "description": "Target group name, which length should be greater than 0 and less than 1025.",
+            "required": true,
+            "type": "string",
+            "maxLength": 1024,
+            "minLength": 1,
+            "pattern": "^(?!\\s+$).+$"
+          },
+          {
+            "in": "path",
+            "name": "userId",
+            "description": "Target user Id.",
+            "required": true,
+            "type": "string",
+            "minLength": 1
+          },
+          {
+            "in": "query",
+            "name": "api-version",
+            "description": "The version of the REST APIs.",
+            "required": true,
+            "type": "string"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "Success"
+          },
+          "default": {
+            "description": "Error response",
+            "schema": {
+              "$ref": "#/definitions/ErrorDetail"
+            },
+            "x-ms-error-response": true,
+            "headers": {
+              "x-ms-error-code": {
+                "x-ms-client-name": "ErrorCode",
+                "type": "string"
+              }
+            }
+          }
+        },
+        "x-ms-examples": {
+          "WebPubSub_AddUserToGroup": {
+            "$ref": "./examples/WebPubSub_AddUserToGroup.json"
+          }
+        }
+      }
+    }
+  },
+  "definitions": {
+    "AddToGroupsRequest": {
+      "description": "The request object containing targets groups and connection filter",
+      "type": "object",
+      "properties": {
+        "groups": {
+          "type": "array",
+          "items": {
+            "type": "string"
+          }
+        },
+        "connectionFilter": {
+          "type": "string"
```

Comment:
Please add a description for this property.

Reply:
updated as suggestion

Reply:
Could you comment with an example filter here? I want to know if user really able to come up with the filter with your description here.